CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 
 ### 💡 Enhancements 💡
 
+- `kubeletstatsreceiver`: Add support for kubeConfig auth_type (#17561)
```

Comment:
This file is auto-generated. Please add an entry in https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/.chloggen